### PR TITLE
Style multiplatform section in landing page

### PR DESCRIPTION
**Pull request by jvaclavik**

### CODE REVIEW

File: apps/landingPage/containers/LandingPage.js [hunk 1]
```diff
@@ -11,31 +11,66 @@ import Features from './Features';
...
+      </ThemeProvider>
+    );
+  }
 }
 
 const Container = styled.div`
   display: flex;
   align-items: center;
   flex-direction: column;
+  position: 'relative';
```
Comment (KarinaDavtyan) [inline apps/landingPage/containers/LandingPage.js]: shouldnt be a string

File: apps/landingPage/containers/LandingPage.js [hunk 2]
```diff
@@ -11,31 +11,66 @@ import Features from './Features';
...
+    }
+  };
+
+  render() {
+    return (
+      <ThemeProvider theme={theme}>
+        <Container>
+          <Menu />
+          <Header />
+          <Features scroll={this.state.scroll} />
```
Comment (KarinaDavtyan) [inline apps/landingPage/containers/LandingPage.js]: could you remove extra line please

File: apps/landingPage/containers/LandingPage.js (hunk 1 above)
Comment (jvaclavik) [reply apps/landingPage/containers/LandingPage.js]: done

File: apps/landingPage/containers/LandingPage.js (hunk 2 above)
Comment (jvaclavik) [reply apps/landingPage/containers/LandingPage.js]: done

File: apps/landingPage/pages/index.js [hunk 3]
```diff
@@ -10,6 +10,10 @@ const GlobalStyle = createGlobalStyle`
     margin: 0;
     -webkit-font-smoothing: antialiased;
     font-family: 'Circular Pro', 'Roboto', sans-serif;
+
```
Comment (KarinaDavtyan) [inline apps/landingPage/pages/index.js]: I think `scroll-snap-type: y proximity;` can go here directly, as it is already `html, body {...}`

Comment (jvaclavik) [reply apps/landingPage/pages/index.js]: done

File: apps/landingPage/components/FloatedPhone.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  return images[0];
+};
+
+export default function FloatedPhone({
+  scroll,
+  featuresHeight,
+  featuresTopPosition,
+}: Props) {
+  const imageIndex =
+    featuresHeight !== 0 && featuresTopPosition !== 0
```
Comment (KarinaDavtyan) [inline apps/landingPage/components/FloatedPhone.js]: Could you breakdown it a little bit please? Its not very readable, or maybe comment it, might be confusing after some time

Comment (jvaclavik) [reply apps/landingPage/components/FloatedPhone.js]: Done

Comment (KarinaDavtyan) [reply apps/landingPage/components/FloatedPhone.js]: Thanks!

File: apps/landingPage/components/FloatedPhone.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+};
+
+export default function FloatedPhone({
+  scroll,
+  featuresHeight,
+  featuresTopPosition,
+}: Props) {
+  const areDimensionsDataKnown =
+    featuresHeight !== 0 && featuresTopPosition !== 0;
+  const imageIndex = areDimensionsDataKnown
```
Comment (KarinaDavtyan) [inline apps/landingPage/components/FloatedPhone.js]:
```suggestion
  const imageIndex = isDimensionsDataKnown
```

**Pull request merged**